src/utils/__tests__/narrow-test.js

```diff
@@ -46,14 +46,7 @@ describe('privateNarrow', () => {
   test('if operator is "pm-with" and only one email, then it is a private narrow', () => {
     expect(isPrivateNarrow([])).toBe(false);
     expect(isPrivateNarrow([{}, {}])).toBe(false);
-    expect(
-      isPrivateNarrow([
-        {
-          operator: 'pm-with',
-          operand: 'bob@example.com',
-        },
-      ]),
-    ).toBe(true);
+    expect(isPrivateNarrow(privateNarrow('bob@example.com'))).toBe(true);
```

Comment:
That is not necessarily better.
Now you are testing two functions. This will fail if our privateNarow is broken, but this is a test for isPrivateNarrow.

Reply:
It is quite reasonable though, to add this as another test.